src/particle/particle/utilities.py

```diff
@@ -43,9 +43,6 @@ def str_with_unc(value, upper, lower=None):
     identical upper/lower bounds. Nicely formats numbers using PDG rule.
     """
 
-    # Only bother with unicode if this is Python 3.
-    pm = u"±" if type(u"") is type("") else "+/-"
```

Comment:
Not sure why it removes random comments that are not touched by the refactor. _Very_ impressive that it can reorder code like this, though. Restored comment.

Reply:
Thanks.